src/main/kotlin/dev/fakek/fakes/FakeCommerce.kt

```diff
@@ -0,0 +1,8 @@
+package dev.fakek.fakes
+
+import dev.fakek.interfaces.Company
+
+/**
+ * FakeCommerce provides a commerce with random information.
+ */
+data class FakeCommerce(val address: FakeAddress, private val fakeCompany: FakeCompany) : Company by fakeCompany
```

Comment:
Nice, I wasn't sure if `by` would work on a data class (makes sense now that I think about it though), really like this solution.